pages/common/eol.md

```diff
@@ -0,0 +1,30 @@
+> CLI to show end-of-life dates (EoLs) for a number of products.
```

Comment:
@kbdharun you accidentally removed the title

```suggestion
# eol

> CLI to show end-of-life dates (EoLs) for a number of products.
```

Reply:
Sorry, Thanks for the fix. I made the suggestions pretty late yesterday.